bigbluebutton-html5/imports/ui/components/modal/simple/component.jsx

```diff
@@ -50,18 +51,29 @@ class ModalSimple extends Component {
         contentLabel={title}
         {...otherProps}
       >
-        <header className={styles.header}>
+        {hideBorder ? <header className={styles.headerNoBorder}>
           <h1 className={styles.title}>{title}</h1>
           <Button
             className={styles.dismiss}
             label={dismiss.label}
-            icon={'close'}
+            icon="close"
             circle
             hideLabel
             onClick={this.handleDismiss}
-            aria-describedby={'modalDismissDescription'}
+            aria-describedby="modalDismissDescription"
           />
-        </header>
+        </header> : <header className={styles.header}>
```

Comment:
Do we need to duplicate this `header` element to change only the style?

Reply:
You should only change the className